L1TriggerConfig/L1ScalesProducers/src/L1CaloHcalScaleConfigOnlineProd.cc

```diff
@@ -82,6 +83,8 @@ L1CaloHcalScaleConfigOnlineProd::L1CaloHcalScaleConfigOnlineProd(
   HcalTopologyMode::Mode mode = HcalTopologyMode::LHC;
   int maxDepthHB = 2;
   int maxDepthHE = 3;
+  edm::LogWarning("") << "Fix how to get HcalTopology correctly";
+  /*
   if( iConfig.exists( "hcalTopologyConstants" ))
```

Comment:
Updated the PR with 3 corrections and rebasing it
________________________________
From: Ianna Osborne [notifications@github.com]
Sent: 30 July 2015 15:01
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-hcx25 Enable runing GEN-SIM-DIGI step with modified Hcal geometry definition (#10353)


In L1TriggerConfig/L1ScalesProducers/src/L1CaloHcalScaleConfigOnlineProd.cc<https://github.com/cms-sw/cmssw/pull/10353#discussion_r35867583>:

> @@ -82,6 +83,8 @@ L1CaloHcalScaleConfigOnlineProd::L1CaloHcalScaleConfigOnlineProd(
>    HcalTopologyMode::Mode mode = HcalTopologyMode::LHC;
>    int maxDepthHB = 2;
>    int maxDepthHE = 3;
> +  edm::LogWarning("") << "Fix how to get HcalTopology correctly";
> +  /*
>    if( iConfig.exists( "hcalTopologyConstants" ))


@mulhearn<https://github.com/mulhearn> - The solution to this is being discussed in L1 software forum.

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/10353/files#r35867583>.